eclair-core/src/test/scala/fr/acinq/eclair/payment/MultiPartHandlerSpec.scala

```diff
@@ -460,4 +467,41 @@ class MultiPartHandlerSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike
     })
   }
 
+  test("KeySend payment in a single HTLC") { f =>
```

Comment:
nit: it would be nice for consistency to harmonize the test names with the rest of the file, for example:

- PaymentHandler should handle single-part keysend payment
- PaymentHandler should reject keysend payment when feature is disabled

Reply:
Done